### PR TITLE
GOV-287.6 Added integration test for key-auth in bulk-processor service

### PR DESCRIPTION
**Pull request by danishjamal104**
Updates in this PR
1. Added integration test for kong key-auth in hello-world service.
2. Created these steps service -> Create Route -> Create Plugin
3. Mapped the consumer to the service.
4. Teardown the entire step.

JIRA -> [GOV-287](https://mifosforge.jira.com/browse/GOV-287)

[GOV-287]: https://mifosforge.jira.com/browse/GOV-287?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by danishjamal104** [comment]
@fynmanoj @avikganguly01 Please review. Thanks :)

**Comment by danishjamal104** [comment]
@ankita10r you can refer this test case for enabling plugin in kong.

**Comment by fynmanoj** [comment]
Please work on the checks

**Review by fynmanoj — APPROVED** [review]

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -83,3 +83,20 @@ identity-account-mapper:
...
+    createKey: /consumers/{username}/key-auth
+    services: /services
+    createRoute: /services/{serviceId}/routes
+    createPlugin: /services/{serviceId}/plugins
+    routes: /routes
+    plugins: /plugins
+  header:
+    apikey: "X-API-KEY"
+  service:
+    host: "ph-ee-connector-bulk.paymenthub.80.svc"
```
Comment (fynmanoj) [inline src/main/resources/application.yaml:100]: why do we need the service here?

**Review by fynmanoj — CHANGES_REQUESTED** [review]
requested clarification

**Pull request merged**